src/Contract/StaticPublishingTrigger.php

```diff
@@ -9,22 +9,21 @@
  */
 interface StaticPublishingTrigger
 {
-
     /**
      * Provides an SS_List of StaticallyPublishable objects which need to be regenerated.
      *
      * @param array $context An associative array with extra engine-specific information.
      *
-     * @return SS_List
+     * @return array|SS_List
```

Comment:
You're forcing an array type in the argument declaration but this PHPDoc claims to accept SS_List as well. I think we need to remove the type declarations...

Reply:
Sorry, bit of confusion - this is the `@returns` so I think it was fine as it was.